test/built-ins/Promise/withResolvers/result.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2023 Peter Klecha. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Promise.withResolvers result is an object with keys "promise", "reject", and "resolve"
+esid: sec-promise.withresolvers
+features: [promise-with-resolvers, Object.hasOwn]
+---*/
+
+
+var instance = Promise.withResolvers();
+
+assert.sameValue(typeof instance, "object");
+assert.notSameValue(instance, null);
+assert(instance intanceof Object);
```

Comment:
Typo here as well: `intanceof` should be `instanceof`.

Reply:
whoops
```suggestion
assert(instance instanceof Object);
```